mk/travis.sh

```diff
@@ -21,30 +21,22 @@ printenv
 
 case $TARGET_X in
 aarch64-unknown-linux-gnu)
-  DL_TARGET=aarch64-linux-gnu
-  DL_DIGEST=b9137008744d9009877f662dbac7481d673cdcb1798e727e325a37c98a0f63da
+  sudo apt-get install --no-install-recommends g++-aarch64-linux-gnu gcc-aarch64-linux-gnu libc6-dev-arm64-cross -y
```

Comment:
>Do we need to use sudo apt-get install to install these, or can we add them as package dependencies the normal way in update_travis_yml.py?

These packages are whitelisted by travis and can be declared on .travis.yml. I can move them there if you prefer. As for update_travis_yml.py the arm/aarch64 are not defined there. Should I take a stab at defining them there?